hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/tenantconfiguration/RepositoryConfigurationView.java

```diff
@@ -125,6 +131,24 @@ UIComponentIdProvider.REPOSITORY_MULTI_ASSIGNMENTS_CHECKBOX, getBinder(),
         gridLayout.addComponent(multiAssignmentsCheckBox, 0, 1);
         gridLayout.addComponent(multiAssignmentsConfigurationItem, 1, 1);
 
+        userConsentCheckBox = FormComponentBuilder.createCheckBox(
+            UIComponentIdProvider.REPOSITORY_USER_CONSENT_CHECKBOX, getBinder(),
+            ProxySystemConfigRepository::isUserConsent, ProxySystemConfigRepository::setUserConsent);
+        userConsentCheckBox.setStyleName(DIST_CHECKBOX_STYLE);
+        userConsentCheckBox.setEnabled(!getBinderBean().isUserConsent());
+        userConsentConfigurationItem.setEnabled(!getBinderBean().isUserConsent());
+        userConsentCheckBox.addValueChangeListener(event -> {
+            actionAutoCloseCheckBox.setEnabled(!event.getValue());
+            actionAutocloseConfigurationItem.setEnabled(!event.getValue());
```

Comment:
I think the auto close checkbox can still be toggled independent of this feature. Only for multi assignment this is required.